Packages/packages-lock.json

```diff
@@ -188,21 +188,14 @@
       "url": "https://packages.unity.com"
     },
     "com.unity.ide.visualstudio": {
-      "version": "2.0.18",
+      "version": "2.0.20",
       "depth": 0,
       "source": "registry",
       "dependencies": {
         "com.unity.test-framework": "1.1.9"
       },
       "url": "https://packages.unity.com"
     },
-    "com.unity.ide.vscode": {
```

Comment:
Same here

Reply:
The reason is explained on the other comment